---
layout: paper
title: "Evolutionary principles underpinning codon usage bias: patterns, functions, and mechanisms"
year: "2024"
shortref: "Cope <i>et al.</i> 2024 "
nickname: exon
journal: "EcoEvoRxiv"
volume: 
issue:
pages: 
authors: "Alexander L. Cope, Michael A. Gilchrist, Premal Shah, Edward Wallace"
image: /assets/images/papers/ASUMDE_model.png 
redirect_from: 
fulltext: https://ecoevorxiv.org/repository/view/7190/
pdflink: 
github: 
pmid: 
pmcid: 
f1000: 
doi: "10.32942/X2802V"
dryad_doi:
figshare_doi: 
zenodo_doi:
altmetric_id: 
category: paper
# Note: 'published' is a Jekyll keyword and does not refer to whether the paper is published, but rather to whether this Markdown should be part of the rendered site.
published: true
preprint: false
embargo: false	
peerreview: true
review: true
tags: [codon usage bias, synonymous mutations, Population genetics, evolution]
---
{% include JB/setup %}

# Abstract 

Synonymous codons are used unevenly despite coding for the same amino acid. Recent work has provided critical insights into the functions, mechanisms, and fitness consequences of codon usage bias and synonymous mutations. However, experiments aimed at understanding the role of synonymous mutations often involve only a small number of reporter genes. How do these observations generalize across genomes, where confounding factors include gene expression and GC content? We propose the following principles for making inferences about the functions, mechanisms, and evolution of codon usage. First, use additive selection-uniform mutation-drift equilibrium as the null model. This evolutionary model explains how codon usage in low-expressed genes is driven by mutation bias and, in high-expressed genes, is driven by selection. It performs well enough to serve as a sensible default for understanding the evolution of codon usage patterns. Second, analyses of codon usage should control for gene expression. The effect of a synonymous change on mRNA translation scales with a gene's total protein production rate such that evolutionary selection on codon usage is strongest in highly-translated genes. Because protein production rate correlates with many other gene features, researchers must control for its effects. Third, researchers must consider mechanistically how codon usage affects biological processes. While correlations between codon usage and other molecular measurements are valuable, proposed mechanistic roles of codon usage must be consistent with established biological mechanisms. In conclusion, the underlying architecture of molecular evolution should be considered before invoking other superficially plausible explanations of codon usage.

