---
layout: paper
title: "The GATA-like transcription factor Gat201 determines alkaline-restricted growth in Cryptococcus neoformans"
year: "2025"
shortref: "Hughes, <i>et al.</i> <i>mSphere</i> 2025"
nickname: gat201-tradeoff
journal: "mSphere"
volume: 
issue:
pages: 
authors: "Hughes ES, Tuck LR, He Z, Ballou ER, Wallace EWJ"
image: /assets/images/papers/default-paper.svg
redirect_from: 
fulltext: https://doi.org/10.1128/msphere.00191-25
pdflink: https://journals.asm.org/doi/reader/10.1128/msphere.00191-25
github: 
pmid: 
pmcid: 
f1000: 
doi: "10.1128/msphere.00191-25"
dryad_doi:
figshare_doi: 
zenodo_doi:
altmetric_id:
category: paper
# Note: 'published' is a Jekyll keyword and does not refer to whether the paper is published, but rather to whether this Markdown should be part of the rendered site.
published: true
preprint: true
embargo: false	
peerreview: false
review: false
tags: [Gat201, transcription, Cryptococcus]
---
{% include JB/setup %}

# Abstract 

The fungus *Cryptococcus neoformans* is an opportunistic human pathogen that causes fatal meningitis through uncontrolled proliferation in host tissues. Evasion of host defenses relies on a protective polysaccharide capsule, regulated, in part, by the GATA-like transcription factor Gat201. Gat201 additionally contributes to virulence through capsule-independent mechanisms. Here, we show that Gat201 affects the proliferation of C. neoformans: in RPMI-1640 cell culture media at an alkaline pH that restricts wild-type cell growth, *gat201∆* strains show increased budding, growth, and viability. RNA-seq analysis shows that Gat201 pathway genes, including co-factors *GAT204* and *LIV3*, are rapidly activated within minutes of inoculating *C. neoformans* in RPMI media, and strains mutated for *GAT204* and, to a lesser extent, LIV3 also show improved growth. The effect of Gat201 on growth is pH-dependent: *gat201∆* cells grow better than wild-type cells at high pH but worse than wild-type cells at neutral pH, in otherwise identical media. Together, this identifies the Gat201 pathway as an alkaline-responsive regulator of proliferation: Gat201 appears to govern an environment-dependent trade-off between proliferation and production of the defensive capsule. Furthermore, evolutionary analysis shows that Gat201 is in a subfamily of GATA-like transcription factors that is conserved within diverse fungi but absent in model yeasts. Together, our findings urge improved understanding of proliferation in diverse environmental niches in order to understand the mechanistic basis of fungal pathogenesis.

# Importance

Infectious microorganisms must adapt to differences between external and host environments in order to colonize and cause disease. *Cryptococcus neoformans* is an encapsulated fungal pathogen that can infect human airways and travel to the brain to cause life-threatening meningitis. The airway is a dynamic environment characterized by nutrient limitation, high temperature (37°C), CO2, and transiently high pH (>8.5). In both the lung and brain, fungal proliferation through budding is a major driver of pathogenesis; however, the regulators of *Cryptococcus* proliferation are poorly understood and distinct from other model yeasts. In this work, we explore how *Cryptococcus* adapts to shifting environments and identify that the transcription factor Gat201, known to regulate capsule production, negatively regulates proliferation under alkaline conditions. Our findings highlight the need for improved understanding of proliferation/adaptation and its regulation in non-model systems.


# Notes

A previous version of this manuscript was the bioRxiv preprint [A trade-off between proliferation and defense in the fungal pathogen Cryptococcus at alkaline pH is controlled by the transcription factor GAT201](https://www.biorxiv.org/content/10.1101/2023.06.14.543486).

Data analysis is spread over multiple github repositories:

- [RNA-seq dataset 1 CryptoWakeupRNASeq](https://github.com/ewallace/CryptoWakeupRNASeq), [doi:10.5281/zenodo.11506094](https://doi.org/10.5281/zenodo.11506094)
- [RNA-seq dataset 2 CryptoGat201RNASeq](https://github.com/ewallace/CryptoGat201RNASeq), [doi:10.5281/zenodo.12207483](https://doi.org/10.5281/zenodo.12207483)
- [Homology analysis, Gat201homology_2022](https://github.com/ewallace/Gat201homology_2022/), [doi:10.5281/zenodo.11506156](https://doi.org/10.5281/zenodo.11506156)
- [Other supplementary data CryptoGat201_2023_suppdata](https://github.com/ewallace/CryptoGat201_2023_suppdata), [doi:10.5281/zenodo.11506135](https://doi.org/10.5281/zenodo.11506135)

